bin/webpack-helpers.js

```diff
@@ -9,10 +9,7 @@ const CHECK_CIRCULAR_DEPS = process.env.CHECK_CIRCULAR_DEPS || false;
 
 // If a package is not available, or missing functionality, in an old but __supported__ version of WordPress, it should be listed here.
 // Some packages are not available in legacy versions of WordPress, so we don't want to extract them.
```

Comment:
This comment block can probably be removed too?

Reply:
Not really, `requiredPackagesInWPLegacy` is still relevant for us to use it when we need it, currently, we just don't need it. So I think it makes sense to leave it there. If the comment was relevant to `@wordpress/compose` then I would delete it.